platform/android/src/run_loop.cpp

```diff
@@ -55,6 +55,15 @@ int looperCallbackDefault(int fd, int, void* data) {
     return 1;
 }
 
+int looperCallbackReadEvent(int fd, int, void* data) {
+    auto runLoopImpl = reinterpret_cast<RunLoop::Impl*>(data);
```

Comment:
is it possible that `looperCallbackReadEvent()` is invoked after `RunLoop::Impl` got deleted?

Reply:
Should not be possible since the HTTPFileSource and curl is cleaned up and deleted before RunLoop deletion.